functions/afaf/afaf.js

```diff
@@ -0,0 +1,25 @@
+const { discordMessageEmbed } = require('../config');
+const { blaccLogo } = require('../constants');
+
+exports.askAnonymously = (question, preferredChannelId, discordClient) => {
+	discordClient.channels.cache
+		.get(preferredChannelId)
+		.send(embedMessage(question));
+};
+exports.getChannel = async (author, currChannel, serverChannels) => {
```

Comment:
Can we make this an "options" object instead of giving the function a list of parameters? I like the scalability of using options for parameters